nfstream/flow.py

```diff
@@ -16,6 +16,8 @@
 from collections import namedtuple
 from math import sqrt
 from .utils import NFEvent
+from inspect import getmembers
```

Comment:
Why you use this package as it is slow as far as I remember.

Reply:
Indeed, I removed it